com.ibm.wala.core/src/testFixtures/java/com/ibm/wala/core/tests/util/WalaTestCase.java

```diff
@@ -63,22 +66,20 @@ protected static void justThisTest(Class<?> testClass) {
   }
 
   protected static String getClasspathEntry(String elt) {
-    StringBuilder result = null;
-    for (String s : System.getProperty("java.class.path").split(File.pathSeparator)) {
-      if (s.contains(elt)) {
-        File e = new File(s);
-        assert e.exists() : elt + " expected to exist";
-        if (e.isDirectory() && !s.endsWith("/")) {
-          s += '/';
-        }
-        if (result == null) {
-          result = new StringBuilder(s);
-        } else {
-          result.append(File.pathSeparator).append(s);
-        }
-      }
-    }
-    assert result != null : "cannot find " + elt;
-    return result.toString();
+    final String normalizedElt = Paths.get(elt).normalize().toString();
```

Comment:
Why is this change in this PR?

Reply:
Per the commit message:

> ### Revamp classpath search to use normalized paths
>
> On Windows with Java 11, the `java.class.path` property contains backslash-escaped backslashes, as in "`C:\\foo\\bar`".  These doubled backslashes interfere with the logic for finding classpath entries that contain the requested element.  Using normalized paths for all comparisons seems to fix the problem, and will likely be more robust in general.

That being said, this change should also work fine under Java 1.8. Therefore I will pull it out into a separate, small pull request that we can merge before the big Java 11 change.